app/code/Magento/Catalog/Api/Data/ProductRenderSearchResultsInterface.php

```diff
@@ -1,16 +1,17 @@
 <?php
 /**
- *
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
 
 namespace Magento\Catalog\Api\Data;
 
+use Magento\Framework\Api\SearchResultsInterface;
+
 /**
  * Dto that holds render information about products
  */
-interface ProductRenderSearchResultsInterface
+interface ProductRenderSearchResultsInterface extends SearchResultsInterface
```

Comment:
Unfortunately, we cannot extend `SearchResultsInterface` since it's backward-incompatible change. Any implementation of `SearchResultsInterface` will be broken after such changes.